Sources/Client/Model/Event.swift

```diff
@@ -295,7 +296,7 @@ public enum Event: Decodable {
         
         func unreadCount() throws -> UnreadCount {
             let channelsUnreadCount = try container.decodeIfPresent(Int.self, forKey: .channelsUnreadCount) ?? 0
-            let messagesUnreadCount = try container.decodeIfPresent(Int.self, forKey: .messagesUnreadCount) ?? 0
+            let messagesUnreadCount = try container.decodeIfPresent(Int.self, forKey: .totalUnreadCount) ?? 0
```

Comment:
I'll rename it to `totalUnreadCount` here. To be honest, I think confusion is unavoidable since backend sends the value as `total_unread_count` but it's actually `messages_unread_count`